k2/python/k2/autograd.py

```diff
@@ -650,6 +657,7 @@ def intersect_dense_pruned(a_fsas: Fsa,
                            b_fsas: DenseFsaVec,
                            search_beam: float,
                            output_beam: float,
+                           allow_partial: bool,
```

Comment:
Please give it a default value so that it won't break existing programs.

Reply:
Also, please add it as the last argument to make it compatible with previous code.

Reply:
fixed.